tests/functional/generate_pages.py

```diff
@@ -101,7 +101,7 @@
 
 ANSWER_LABEL_GETTER = Template(
     r"""  ${answerName}Label() {
-    return `[for=${answerId}]`;
+    return `#${answerId}-label`;
```

Comment:
I don't see how adding the id helps here? The original seems more concise and we didn't have an issue with answer label chains.

Reply:
Im making changes in here because I needed to update to version 18 of the Design System and there are some changes to labels in there, not just because of the chained selector stuff. As part of this I have also added ids to all the labels, so thought that was an easy and specific way to select them, now that they have ids, rather than using the `for` attribute. Having said that I'm still working on this in order to get the functional tests to pass.

Reply:
Does v18 of the design system no longer allow selection using `for`?

Reply:
I kind of feel that keeping the use of `for` and not having to specify the id across all answer templates/rendered pages is the preferable approach here, especially as we're only really making changes for the purposes of our tests.

Reply:
Ok no worries ill change it back